processor/src/operations/crypto_ops.rs

```diff
@@ -51,10 +51,10 @@ impl Process {
     /// To perform the operation we do the following:
     /// 1. Look up the Merkle path in the advice provider for the specified tree root.
     /// 2. Use the hasher to compute the root of the Merkle path for the specified node.
-    /// 3. Replace the node value with the computed root.
+    /// 3. Copy the stack state over to the next clock cycle with no changes.
     ///
     /// If the correct Merkle path was provided, the computed root and the provided root must be
-    /// the same. This can be checked via subsequent operations.
+    /// the same. This is checked via an assert statement.
```

Comment:
Let's add "Panics" section and put info that we panic in case roots are not the same there.

Reply:
Done